tests/suite/foundations/calc.typ

```diff
@@ -126,11 +126,16 @@
 // Test the `quo` function.
 #test(calc.quo(1, 1), 1)
 #test(calc.quo(5, 3), 1)
-#test(calc.quo(5, -3), -1)
+#test(calc.quo(5, -3), -2)
+#test(calc.quo(-5, 3), -2)
+#test(calc.quo(-5, -3), 1)
+#test(calc.quo(6, -3), -2)
```

Comment:
For good measure, I think the example in the linked issue (with 0 and -1, both should be the same now) should also be added.